src/node.cc

```diff
@@ -3088,6 +3054,17 @@ void RegisterBuiltinModules() {
 #undef V
 }
 
+const std::string GetOpenSSLVersion() {
+  // sample openssl version string format
+  // for reference: "OpenSSL 1.1.0i  14 Aug 2018"
+  std::string ssl(OPENSSL_VERSION_TEXT);
+  size_t first = ssl.find(" ");
+  size_t second = ssl.find(" ", first + 1);
+  CHECK_GT(second, first);
+  std::string result = ssl.substr(first + 1, second - first - 1);
+  return result;
```

Comment:
I think you can just `return ssl.substr(…);`?

Reply:
👍 since in C++14 it's not guaranteed to elide